test/extension.test.ts

```diff
@@ -762,3 +780,41 @@ suite('Compilation info', () => {
     expect(info.compiler).to.eq('cl.exe');
   });
 });
+
+suite('CMake System tests', () => {
+  (process.env.HasVs == 'true' ? suite.skip : suite)('No present kit',() => {
+        let path_backup = "";
+        suiteSetup(()=>{
+          clearExistingKitConfigurationFile();
+
+          // Test will use path to scan for compilers
+          // with no path content there is no compiler found
+          path_backup = process.env.PATH!;
+          process.env.PATH = "";
```

Comment:
I think we should use `''` for all strings.

Reply:
Done